recipes/stacks/1.37/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: stacks
+  version: "1.37"
+build:
+  number: 1
+  skip: True # [osx]
```

Comment:
If you skip osx you probably do not need to include llvm.
The build number should be 0 if this is the first version.